versions-shipped.json

```diff
@@ -1 +1,187 @@
-[]
+[
```

Comment:
the `version*` files should not be committed

Reply:
well noted